docs/docs-content/clusters/pcg/deploy-pcg/vmware.md

```diff
@@ -0,0 +1,550 @@
+
+For example, assume a vCenter environment contains three compute clusters, cluster-1, cluster-2, and cluster-3. To
+support this environment you create the tag categories `k8s-region` and `k8s-zone`. The `k8s-region` is assigned to the
+datacenter, and the `k8s-zone` tag is assigned to the compute clusters.
+
+The following table lists the tag values for the data center and compute clusters.
+
+| **vSphere Object** | **Assigned Name** | **Tag Category** | **Tag Value** |
+| ------------------ | ----------------- | ---------------- | ------------- |
+| **Datacenter**     | dc-1              | k8s-region       | region1       |
+| **Cluster**        | cluster-1         | k8s-zone         | az1           |
+| **Cluster**        | cluster-2         | k8s-zone         | az2           |
+| **Cluster**        | cluster-3         | k8s-zone         | az3           |
+
+Create a tag category and tag values for each datacenter and cluster in your environment. Use the tag categories to
+create zones. Use a name that is meaningful and that complies with the tag requirements listed in the following section.
+
+#### Tag Requirements
+
+The following requirements apply to tags:
+
+- A valid tag must consist of alphanumeric characters.
+
+- The tag must start and end with an alphanumeric characters.
+
+- The regex used for tag validation is `(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?`
+
+## Deploy PCG
+
+1.  In an x86 Linux host with the Palette CLI installed, open up a terminal session.
+
+2.  Issue the following command to authenticate with Palette. When prompted, enter the required information. Refer to
+    the table below for information about each parameter.
+
+    ```shell
+    palette login
+    ```
+
+    | **Parameter**                  | **Description**                                                                                                                                                                                                                                                   |
+    | :----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | **Spectro Cloud Console**      | Enter the Palette endpoint URL. When using the Palette SaaS service, enter `https://console.spectrocloud.com`. When using a self-hosted instance of Palette, enter the URL for that instance.                                                                     |
+    | **Allow Insecure Connection**  | Enabling this option bypasses x509 server Certificate Authority (CA) verification. Enter `y` if you are using a self-hosted Palette, or VerteX instance with self-signed TLS certificates and need to provide the file path to instance CA. Otherwise, enter `n`. |
```

Comment:
```suggestion
    | **Allow Insecure Connection**  | Enabling this option bypasses x509 server Certificate Authority (CA) verification. Enter `y` if you are using a self-hosted Palette or VerteX instance with self-signed TLS certificates and need to provide a file path to the instance CA. Otherwise, enter `n`. |
```